data-products/src/recommendation_api/sql/scheduled_syndicated.sql

```diff
@@ -1,11 +1,13 @@
 SELECT
-    APPROVED_CORPUS_ITEM_EXTERNAL_ID as "ID",
-    TOPIC as "TOPIC",
-    PUBLISHER AS "PUBLISHER"
-FROM "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS"
-WHERE SCHEDULED_SURFACE_ID = %(SCHEDULED_SURFACE_ID)s
-AND approved_corpus_item_external_id NOT IN ('904bc245-54de-412a-a8af-c13eb0a7c15c') -- See #incidents on 2023-03-21 and DENG-4478
-AND IS_SYNDICATED
-AND SCHEDULED_CORPUS_ITEM_SCHEDULED_AT BETWEEN DATEADD(day, %(MAX_AGE_DAYS)s, CURRENT_DATE) AND CURRENT_DATE
-QUALIFY row_number() OVER (PARTITION BY APPROVED_CORPUS_ITEM_EXTERNAL_ID ORDER BY SCHEDULED_CORPUS_ITEM_SCHEDULED_AT DESC) = 1
-ORDER BY SCHEDULED_CORPUS_ITEM_SCHEDULED_AT DESC
+    s.APPROVED_CORPUS_ITEM_EXTERNAL_ID as "ID",
+    i.TOPIC as "TOPIC",
+    i.PUBLISHER AS "PUBLISHER"
+FROM "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS" AS s
+LEFT JOIN "ANALYTICS"."DBT"."CORPUS_ITEMS_UPDATED" AS i
+ON s.APPROVED_CORPUS_ITEM_EXTERNAL_ID = i.APPROVED_CORPUS_ITEM_EXTERNAL_ID
+WHERE s.SCHEDULED_SURFACE_ID = %(SCHEDULED_SURFACE_ID)s
+AND i.TOPIC IS NOT NULL
+AND s.IS_SYNDICATED
```

Comment:
```suggestion
AND i.IS_SYNDICATED
```
`IS_SYNDICATED` happens to be an attribute of corpus item. This way we are picking the most recent update (if any)